src/NuGet.Core/NuGet.Common/Properties/AssemblyInfo.cs

```diff
@@ -1,6 +1,13 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Runtime.CompilerServices;
 
 [assembly: CLSCompliant(true)]
+
+#if SIGNED_BUILD
+[assembly: InternalsVisibleTo("NuGet.Common.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
+#else
+[assembly: InternalsVisibleTo("NuGet.Common.Test")]
```

Comment:
We actually don't need this anymore. 
Delay signing is enabled by default :)

Reply:
I don't like that signing is enabled by default. My memory is that we did it for ngen, but we only ngen on CI builds, not local builds, so I don't understand why we're delay signing local builds. Forcing all developers to trust the fake/temporary certificate feels really wrong to me.

Reply:
That's cause of ProvideCodeBaseAttribute. Don't necessarily see it as wrong, though :)